modules/videoio/src/cap_msmf.cpp

```diff
@@ -1189,6 +1193,11 @@ bool CvCapture_MSMF::open(int index, const cv::VideoCaptureParameters* params)
         close();
         return false;
     }
+    if (isOpen)
+    {
+        if (audioStream != -1)
+            checkAudioProperties();
```

Comment:
> checkAudioProperties();

We should not ignore return value.
Function returns false on errors. Perhaps we should `.close` capture object in that case.

When is `configureAudioOutput()` called in this flow?

Reply:
> 

In open, first called configureOutput, then inside this function called configureAudioOutput. If I understood the question correctly

Reply:
My bad. This note is about `setAudioProperties()` call above (line 1154)